Oracle_CQRS/src/main/java/org/example/oracle_cqrs/OracleCqrsApplication.java

```diff
@@ -0,0 +1,36 @@
+package org.example.oracle_cqrs;
+
+import jakarta.jms.ConnectionFactory;
+import jakarta.jms.JMSException;
+import oracle.jakarta.jms.AQjmsFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.jms.support.converter.MappingJackson2MessageConverter;
+import org.springframework.jms.support.converter.MessageType;
+
+import javax.sql.DataSource;
+
+
+@SpringBootApplication
+public class OracleCqrsApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(OracleCqrsApplication.class, args);
+    }
+
+
+    @Bean
+    public ConnectionFactory aqJmsConnectionFactory(DataSource ds) throws JMSException {
+        return AQjmsFactory.getConnectionFactory(ds);
+    }
+
+
+    @Bean
+    public MappingJackson2MessageConverter jacksonJmsMessageConverter() {
```

Comment:
Something for us to look into (I may do this): How can we make a JMS message converter that uses OSON (Oracle binary JSON format)?